LayoutTests/platform/ios-simulator/editing/spelling/document-markers-remain-after-removing-text-expected.txt

```diff
@@ -0,0 +1,9 @@
+layer at (0,0) size 800x600
+  RenderView at (0,0) size 800x600
+layer at (0,0) size 800x38
+  RenderBlock {HTML} at (0,0) size 800x38
+    RenderBody {BODY} at (8,8) size 784x22
+      RenderBlock {DIV} at (0,0) size 302x22 [border: (1px solid #000000)]
+        RenderText {#text} at (1,1) size 105x19
+          text run at (1,1) width 101: "aaaa bbbb dddd"
+          text run at (101,1) width 5: " "
```

Comment:
This result doesn't tell us anything. We either need a reference or some assertions.

Reply:
Also, we should avoid platform specific results if possible.